substrate/frame/examples/tasks/src/lib.rs

```diff
@@ -59,9 +63,33 @@ pub mod pallet {
 		}
 	}
 
+	#[pallet::hooks]
+	impl<T: Config> Hooks<BlockNumberFor<T>> for Pallet<T> {
+		#[cfg(feature = "experimental")]
+		fn offchain_worker(_block_number: BlockNumberFor<T>) {
+			if let Some(key) = Numbers::<T>::iter_keys().next() {
+				// Create a valid task
+				let task = Task::<T>::AddNumberIntoTotal { i: key };
+				let runtime_task = <T as Config>::RuntimeTask::from(task);
+				let call = frame_system::Call::<T>::do_task { task: runtime_task.into() };
+
+				// Submit the task as an unsigned transaction
+				SubmitTransaction::<T, frame_system::Call<T>>::submit_unsigned_transaction(
+					call.into(),
+				)
+				.map_err(|()| "Unable to submit unsigned transaction.")
+				.unwrap();
```

Comment:
```suggestion
				.defensive();
```

Reply:
Defensive adds a log in prod and panics in tests :)

Reply:
Oh, I didn't know. Thanks :))